src/bootstrap/angular.bootstrap.js

```diff
@@ -37,14 +42,28 @@ module.exports = function(dependentModulesMap) {
     }]);
   }
 
-  function onChangeLiveEditAngular(event, targetCSSClassName, doc) {
-    var elem = doc.querySelector(targetCSSClassName);
-
-    angularCompile(elem, event.srcElement.value);
+  function onChangeLiveEditAngular(target, value) {
+    angularCompile(target, value);
   }
 
-  window.onChangeLiveEditAngular = onChangeLiveEditAngular;
-  // debounce(onChangeLiveEditAngular, 200);
+  // Create a new Code Mirror instance for each editor
+  const liveEditors = document.getElementsByClassName('live-editor');
```

Comment:
As a solution to the first point (above), what if each live-edit block for each framework was given the name "live-edit-{frameworkName}"?